src/test/java/de/fraunhofer/aisec/cpg/enhancements/DFGTest.java

```diff
@@ -350,13 +352,149 @@ void testDelayedAssignment() throws Exception {
+            false);
+    assertTrue(dfgNodes.contains(l0));
+    assertTrue(dfgNodes.contains(l1));
+    assertTrue(dfgNodes.contains(l2));
+
+    assertTrue(!dfgNodes.contains(l3));
+  }
+
+  /**
+   * Gets Integer Literal from the List of nodes to simplify testsyntax. The Literal is expected to
+   * be contained in the list and the function will throw an {@link IndexOutOfBoundsException}
+   * otherwise.
+   *
+   * @param nodes - The list of nodes to filter for the Literal.
+   * @param v - The integer value expected from the Literal.
+   * @return The Literal with the specified value.
+   */
+  private Literal getLiteral(List<Node> nodes, int v) {
+    return (Literal)
+        TestUtils.findByPredicate(
+                nodes,
+                n -> n instanceof Literal && ((Literal) n).getValue().equals(Integer.valueOf(v)))
+            .get(0);
+  }
+
+  @Test
+  void testSensitivityWithLabels() throws Exception {
+    Path topLevel = Path.of("src", "test", "resources", "dfg");
+    TranslationUnitDeclaration result =
+        TestUtils.analyze(List.of(topLevel.resolve("LoopDFGs.java").toFile()), topLevel, true)
+            .get(0);
+    MethodDeclaration looping =
+        TestUtils.getSubnodeOfTypeWithName(result, MethodDeclaration.class, "labeledBreakContinue");
+
+    List<Node> methodNodes = SubgraphWalker.flattenAST(looping);
+    Literal l0 = getLiteral(methodNodes, 0);
+    Literal l1 = getLiteral(methodNodes, 1);
+    Literal l2 = getLiteral(methodNodes, 2);
+    Literal l3 = getLiteral(methodNodes, 3);
+    Literal l4 = getLiteral(methodNodes, 4);
+    List<Node> calls =
+        TestUtils.findByPredicate(
+            SubgraphWalker.flattenAST(looping),
+            n -> n instanceof CallExpression && n.getName().equals("println"));
+    calls.sort(new NodeComparator());
+    Set<Node> dfgNodesA0 =
+        flattenDFGGraph(
+            TestUtils.getSubnodeOfTypeWithName(
+                calls.get(0), DeclaredReferenceExpression.class, "a"),
+            false);
+
+    Set<Node> dfgNodesA1 =
+        flattenDFGGraph(
+            TestUtils.getSubnodeOfTypeWithName(
+                calls.get(1), DeclaredReferenceExpression.class, "a"),
+            false);
+    Set<Node> dfgNodesA2 =
+        flattenDFGGraph(
+            TestUtils.getSubnodeOfTypeWithName(
+                calls.get(2), DeclaredReferenceExpression.class, "a"),
+            false);
+
+    assertTrue(dfgNodesA0.contains(l0));
+    assertTrue(dfgNodesA0.contains(l1));
+    assertTrue(dfgNodesA0.contains(l3));
+    assertTrue(!dfgNodesA0.contains(l4));
+
+    assertTrue(dfgNodesA1.contains(l0));
+    assertTrue(dfgNodesA1.contains(l1));
+    assertTrue(dfgNodesA1.contains(l3));
+    assertTrue(!dfgNodesA1.contains(l4));
+
+    assertTrue(dfgNodesA2.contains(l0));
+    assertTrue(dfgNodesA2.contains(l1));
+    assertTrue(dfgNodesA2.contains(l2));
+    assertTrue(dfgNodesA2.contains(l3));
+    assertTrue(!dfgNodesA2.contains(l4));
```

Comment:
same here

Reply:
Typed assertFalse and my IDE didn't autocomplete it, so it was more because of my laziness that I did not investigate on that.